### PR TITLE
Remodels king goat ruin

### PR DESCRIPTION
**Pull request by MMMiracles**
## About The Pull Request

- Remakes both the entrance and main arena for the king goat
- Adds indestructible carpet
- Adds goat harem

## Why It's Good For The Game
Striding for some standard of mapping when it comes to ruins.

## Changelog
:cl: MMMiracles
tweak: Goat King ruin has been remodeled to be more aesthetically pleasing
/:cl:

image
image

### CODE REVIEW

**Comment by wesoda25** [comment]
thanks for making me read the phrase “goat harem”

**Comment by Pumpkinoe** [comment]
wow it looks actually kind of good wtf

**Comment by ATH1909** [comment]
~~Why are there only two beds?~~

**Comment by ATH1909** [comment]
Can you fire projectiles (from, say, an emitter) through the blue portal and have them hit the goat king for a cheap kill?

File: code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm [hunk 1]
```diff
@@ -107,6 +107,17 @@ Difficulty: Insanely Hard
 		return A
 	return ..()
 
+/mob/living/simple_animal/hostile/retaliate/goat/harem
+	name = "goat with a wig"
+	desc = "A very ...handsome beast?"
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm:112]: the space should be after the ellipsis, not before it

File: code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm [hunk 2]
```diff
@@ -107,6 +107,17 @@ Difficulty: Insanely Hard
 		return A
 	return ..()
 
+/mob/living/simple_animal/hostile/retaliate/goat/harem
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm:110]: can you make this a subtype of guard golems

**Comment by Coastfront** [comment]
I can't wait to have graphic, long lasting sex with his goat harem ontop of his mutilated corpse.

File: code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm:110]: :(

**Pull request merged**